runtime/src/types.rs

```diff
@@ -1,35 +1,35 @@
-use crate::{metadata, Config, InterBtcRuntime, RuntimeCurrencyInfo, SS58_PREFIX};
+use crate::{metadata, utils::signer::PairSigner, Config, InterBtcRuntime, RuntimeCurrencyInfo};
 pub use metadata_aliases::*;
-pub use subxt::ext::sp_core::{crypto::Ss58Codec, sr25519::Pair as KeyPair};
-use subxt::{
-    metadata::DecodeStaticType,
-    storage::{address::Yes, StaticStorageAddress},
-};
-
 pub use primitives::{
     CurrencyId,
     CurrencyId::{ForeignAsset, LendToken, Token},
     TokenSymbol::{self, DOT, IBTC, INTR, KBTC, KINT, KSM},
 };
+pub use sp_core::sr25519::Pair as KeyPair;
+use subxt::storage::{address::Yes, Address};
 
+pub use crate::utils::{account_id as utils_accountid, multi_signature};
 pub use currency_id::CurrencyIdExt;
 pub use h256_le::RichH256Le;
 pub use module_btc_relay::{RichBlockHeader, MAIN_CHAIN_ID};
 
-pub type AccountId = subxt::ext::sp_runtime::AccountId32;
+//ToDo: Fork account id in utils there are some traits there in compat where it can cause errors to us
+// also multi address and what not :)
+pub type AccountId = utils_accountid::AccountId32;
+pub type MultiSignature = multi_signature::MultiSignature;
 pub type Balance = primitives::Balance;
 pub type Index = u32;
 pub type BlockNumber = u32;
-pub type H160 = subxt::ext::sp_core::H160;
-pub type H256 = subxt::ext::sp_core::H256;
-pub type U256 = subxt::ext::sp_core::U256;
+pub type H160 = sp_core::H160;
```

Comment:
Does subxt not still re-export sp-core?

Reply:
we can't use `subxt::ext:: sp_core` it's feature gated by `substrate-compat` feature flag